docs/user_guide/datasets/acs_income.rst

```diff
@@ -160,6 +160,13 @@ classification task.
    *  - PINCP
       - Total annual income per person, denoted as an integer ranging from 104 to 1,423,000.
 
+.. _using_acsincome_dataset:
+
+Using the dataset
+^^^^^^^^^^^^^^^^^
+The dataset can be loaded via the :func:`fairlearn.datasets.fetch_acs_income`
+function. By default, the dataset is returned as a :class:`pandas.DataFrame`.
```

Comment:
Thanks for the PR. I don't think this requires its own section. This deserves to be after the top most paragraph.

Reply:
I made it consistent with [https://fairlearn.org/v0.10/user_guide/datasets/diabetes_hospital_data.html](url). If it shouldn't have a section, then the other pages maybe should also be changed.

Reply:
I see. Then we can leave it as is. I do think this kind of information could be more on the top, but happy to leave it here.